test/parallel/test-fs-close-error.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+common.expectsError(
+  () => {
+    fs.closeSync('test');
+  },
+  {
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
+    message: 'The "fd" argument must be of type integer'
+  }
+);
+
+common.expectsError(
+  () => {
+    fs.close('test');
+  },
+  {
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
+    message: 'The "fd" argument must be of type integer'
+  }
+);
+
+assert.throws(
+  () => { fs.closeSync(-1); },
+  (err) => {
+    assert.strictEqual(err.code, 'EBADF');
```

Comment:
The current error does not include any information about the `fd` being closed (I think it's also the same for other fs APIs that takes fd as arguments). To improve this properly we would need to update the `After` callback so that async APIs can be consistent with the synchronous APIs. Since `After` is also used by functions not migrated yet, I would like to update this bit when all the fs errors are migrated to the JS layer.